rebar.config

```diff
@@ -177,10 +177,11 @@
 {cover_enabled, true}.
 {cover_print_enabled, true}.
 {cover_export_enabled, true}.
-{coveralls_coverdata, "_build/**/cover/*.coverdata"}.
+{coveralls_coverdata, "/tmp/mongoose_combined.coverdata"}.
 {coveralls_service_name, "circle-ci"}.
 
 {codecov_opts,
  [
-  {path, ["_build/**/cover"]}
+  %% Assuming /tmp/mongoose_combined.coverdata
+  {path, ["/tmp"]}
```

Comment:
As I understand, we leave additional trash on the host machine's `/tmp` directory

Reply:
yeah, trash was there before, we always were writing mongoose_combined there.
There is a set of files in /tmp, including markdown comments.

Can be addressed in a separate PR, I want to merge elasticsearch fix asap.